user_coffee_shop_query.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'distance_calculator'
+
+class UserCoffeeShopQuery
+  attr_reader :distances
+
+  def initialize(coffee_shops, user)
+    @coffee_shops = coffee_shops
+    @user = user
+  end
+
+  def calculate_distances
```

Comment:
If this intended to be object internal method, it should be marked as `private`

Reply:
Absolutely right. My bad :+1:

Reply:
I knew it was just unintentional mistake:)